20_aws_networking/aws_ssm.tf

```diff
@@ -0,0 +1,37 @@
+locals {
```

Comment:
WTF is this, and why you need this??

Reply:
This needs to communicate with private instances via Session Manager. Without that stuff it works only for public ones